app/waf/src/services/do.service.ts

```diff
@@ -155,6 +155,25 @@ export class OnboardingManager {
       }
     },
 
+    ntp: (
+      target: TypeDOClassDeclaration['Common'],
+      obData: Adc,
+      additionalInfo?: object,
+    ) => {
+      try {
+        let ntpData = {
+          class: 'NTP',
+          servers: ['0.pool.ntp.org', '1.pool.ntp.org', '2.pool.ntp.org'],
+          timezone: 'UTC',
+        };
+        this.logger.debug('Add new ntp.');
+        return Object.assign(target, {myNtp: ntpData});
+      } catch (error) {
+        this.logger.debug('No ntp found.');
+        return target;
```

Comment:
does not need to throw, right?

Reply:
Yeah, make it behaves like the onboarding body assembling on ntp did not happen.

Reply:
But should make the error.message print into debug logs. 
Will do it in another PR.